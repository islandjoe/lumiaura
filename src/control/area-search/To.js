import React from 'react'
import '../../layout/JobType.css'

const toLoc =(props)=> {
  return (
    <div className="ui input">
      <input
          type="text"
          placeholder="To..."
          onKeyPress={ props.locationTo } />
    </div>
  )
}

export default toLoc