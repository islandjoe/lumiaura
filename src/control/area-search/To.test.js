import React from 'react'
import renderer from 'react-test-renderer'
import To from './To'

describe('<To/>', ()=> {
  test('should render', ()=> {
    const c = renderer
      .create(<To/>)
      .toJSON()

    expect(c).toMatchSnapshot()
  })
})