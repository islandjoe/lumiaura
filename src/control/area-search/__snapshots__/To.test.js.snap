// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<To/> should render 1`] = `
<div
  className="ui input"
>
  <input
    placeholder="To..."
    type="text"
  />
</div>
`;
